pkg/greenlake/login.go

```diff
@@ -30,7 +31,15 @@ func newLoginCommand() *cobra.Command {
 			return validateArgs(&opts)
 		},
 		RunE: func(_ *cobra.Command, _ []string) error {
-			return runLogin(&opts)
+			err := runLogin(&opts)
+			if analytics.CheckGoogleAnalytics() {
+				err1 := runAnalytics()
```

Comment:
This seems wrong.... if we can't post data we don't want to fail the command.  Not every CLI will have access to the internet.  We don't' want to fail the command if we can't post data to GA